filter.js

```diff
@@ -38,9 +41,9 @@ function Filter(inputTree, options) {
   this._destFilePathCache = Object.create(null);
 }
 
-Filter.prototype.rebuild = function() {
+Filter.prototype.build = function build() {
   var self = this;
-  var srcDir = this.inputPath;
+  var srcDir = this.inputPaths[0];
```

Comment:
No, because there is guaranteed to be [exactly one](https://github.com/joliss/cauliflower-filter/blob/762bec86f6508fdb9ff992d66205b40be7c15278/filter.js#L25).